plugin/src/main/java/com/denizenscript/denizen/objects/properties/entity/EntityVariant.java

```diff
@@ -0,0 +1,47 @@
+package com.denizenscript.denizen.objects.properties.entity;
+
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizen.utilities.Utilities;
+import com.denizenscript.denizencore.objects.Mechanism;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import org.bukkit.entity.Wolf;
+
+public class EntityVariant extends EntityProperty<ElementTag> {
+
+    // <--[property]
+    // @object EntityTag
+    // @name variant
+    // @input ElementTag
+    // @description
+    // Controls which variant a wolf is.
+    // Valid variants: ASHEN, BLACK, CHESTNUT, PALE, RUSTY, SNOWY, SPOTTED, STRIPED, WOODS
```

Comment:
I'd just link the JD instead of manually listing them, saves us having to update the list if it changes (`<@link url https://XXX>`)